File: programme/check-even-or-odd-string/check-even-or-odd-string.c
// Write a programme to check even or odd string

#include <stdio.h>
#include <string.h>

void check_even(char *input)
{
  if (strlen(input) % 2 == 0)
  {
    printf("Even String\n");
  }
  else
  {
    printf("Odd String\n");
  }
}

int main()
{
  char *first = "hiithere";
  printf("Input  : %s\n", first);
  printf("Output : ");
  check_even(first);

  return 0;
}

File: programme/check-even-or-odd-string/check-even-or-odd-string.js
// Write a programme to check even or odd string

function checkEven(input) {
  return input.length % 2 == 0 ? "Even String" : "Odd String";
}

const first = "hiithere";
console.log("Input  : " + first);
console.log("Output : " + checkEven(first));
